### ISSUE TITLE
0000314:
default containers

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#250]
**Reported by pschuele on 2 Sep 2008 13:04**

implement new default container concept

**Comment by pschuele** [comment]
imho no longer relevant

**Comment by Gloirin** [comment]
Depends on #1276

**Comment by Gloirin** [comment]
Depends on #1277
